### PR TITLE
New package: gef-2020.06

### PR DESCRIPTION
**Pull request by abenson**
1. I don't like that keystone python bindings require patching
2. I've already submitted an issue upstream to not use `-` in versions
3. Even if `gef` drops off this PR (understandable) the new packages and changes to existing packages are necessary for it.

### CODE REVIEW

**Comment by fosslinux** [comment]
IMO, if they refuse to not use `-` in versions, replacing `-` with `.` would be acceptable.

**Comment by abenson** [comment]
Apparently that was the first time they did, and they did answer my PR about the future. Doesn't look like they'll fix that, but we'll see.

File: srcpkgs/keystone/patches/fix-lib-name.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- bindings/python/keystone/keystone.py.orig
++++ bindings/python/keystone/keystone.py
+@@ -20,7 +20,7 @@
+     __file__ = inspect.getfile(inspect.currentframe())
+ 
+ _lib_path = split(__file__)[0]
+-_all_libs = ('keystone.dll', 'libkeystone.so', 'libkeystone.dylib')
++_all_libs = ('keystone.dll', 'libkeystone.so.0', 'libkeystone.dylib')
```
Comment (abenson) [inline srcpkgs/keystone/patches/fix-lib-name.patch:8]: I've submitted a patch to upstream, looking to see what they say. That patch is better, so we'll see.

**Comment by abenson** [comment]
1. keystone bindings have been a patched upstream.
2. new release without `-` for gef has been released
3. I feel comfortable with this now

**Pull request merged**